test/unit_test/fft_test_utils.hpp

```diff
@@ -425,7 +425,20 @@ void run_test(const test_params& params) {
   decltype(host_reference_output) host_output(desc.get_output_count(params.dir), padding_value);
   decltype(host_reference_output_imag) host_output_imag(
       Storage == complex_storage::SPLIT_COMPLEX ? desc.get_output_count(params.dir) : 0, padding_value);
-  double tolerance = 1e-3;
+  double tolerance = 5e-2;
```

Comment:
Why do we need to increase the tolerance now ?

Reply:
5e-2 is a big tolerance value, so much so that legitimate incorrect outputs due to some bugs can be off by 1e-2 from their correct value.

Reply:
fixed to use max theoretical error for Cooley-Tukey